assets/src/edit-story/app/prepublish/constants.js

```diff
@@ -53,208 +53,256 @@ export const PRE_PUBLISH_MESSAGE_TYPES = {
 export const MESSAGES = {
   CRITICAL_METADATA: {
     MISSING_TITLE: {
-      MAIN_TEXT: __('Missing story title', 'web-stories'),
-      HELPER_TEXT: sprintf(
-        /* translators: 1: minimum story title length in words. 2: maximum story title length in characters. */
-        __(
-          'Keep the title clear and clean, ideally under %1$d words in less than %2$d characters.',
-          'web-stories'
-        ),
-        MAX_STORY_TITLE_LENGTH_WORDS,
-        MAX_STORY_TITLE_LENGTH_CHARS
+      MAIN_TEXT: __('Add story title', 'web-stories'),
+      HELPER_TEXT: (
+        <TranslateWithMarkup>
+          {sprintf(
+            /* translators: 1: minimum story title length in words. 2: maximum story title length in characters. */
+            __(
+              "- Keep under %1$d words<br/>- Don't exceed %2$d characters",
+              'web-stories'
+            ),
+            MAX_STORY_TITLE_LENGTH_WORDS,
+            MAX_STORY_TITLE_LENGTH_CHARS
+          )}
```

Comment:
First, these hyphens everywhere should really not be part of the translatable strings itself. They should be really be done in CSS, for example by actually using `li` items (because it's a list of recommendations after all) and using `list-style-type: '-'`. It does not make sense for translators to translate a hyphen.

Second, this one should be split up into two and use `_n()` instead of `__()`. That's because in some languages, the plurals of `words` and `characters` needs to be translated differently depending on the number being referenced.

Example:

```js
sprintf(
  _n( 'Keep under %d word', 'Keep under %d words', MAX_STORY_TITLE_LENGTH_WORDS, 'web-stories' ),
  MAX_STORY_TITLE_LENGTH_WORDS
);

sprintf(
  _n( 'Don't exceed %d character', 'Don't exceed %d characters', MAX_STORY_TITLE_LENGTH_CHARS, 'web-stories' ),
  MAX_STORY_TITLE_LENGTH_CHARS
);
```

So basically like this:

```js
<ul>
	  <li>
		    {sprintf(
		      _n( 'Keep under %d word', 'Keep under %d words', MAX_STORY_TITLE_LENGTH_WORDS, 'web-stories' ),
		      MAX_STORY_TITLE_LENGTH_WORDS
		    )}
	  </li>
	  <li>
		    {sprintf(
		      _n( 'Don't exceed %d character', 'Don't exceed %d characters', MAX_STORY_TITLE_LENGTH_CHARS, 'web-stories' ),
		      MAX_STORY_TITLE_LENGTH_CHARS
		    )}
	  </li>
</ul>
```

Reply:
The same goes for all other similar changes  here  of course

Reply:
Good shout! I'll get these fixed 👍